lib/msal-common/src/telemetry/server/ServerTelemetryManager.ts

```diff
@@ -158,4 +165,70 @@ export class ServerTelemetryManager {
 
         return maxErrors;
     }
+
+    /**
+     * Get the region discovery fields
+     * 
+     * @returns string
+     */
+    getRegionDiscoveryFields(): string {
+        const regionDiscoveryFields: string[] = [];
+
+        regionDiscoveryFields.push(this.regionUsed ? this.regionUsed : "");
+        regionDiscoveryFields.push(this.getRegionSourceValue(this.regionSource));
+        regionDiscoveryFields.push(this.getRegionOutcomeValue(this.regionOutcome));
+
+        return regionDiscoveryFields.join(",");
+    }
+
+    /**
+     * Get the header value for the region source
+     * @param regionSource
+     * @returns string
+     */
+    getRegionSourceValue(regionSource: RegionDiscoverySources | undefined): string {
+        if (!regionSource) return "";
+
+        return regionSource;
```

Comment:
Why is this function needed? Are we just returning the value passed in?

Reply:
Good catch, I have removed it.